pages/_slug/index.vue

```diff
@@ -22,8 +22,9 @@
             :url="card.fields.url"
             :image-url="card.fields.image.fields.file.url"
           />
-        </ul>
+        </b-card-group>
       </div>
+
```

Comment:
when running via 'npm run dev' this line throws a linting error, it expects there not to be an empty line here.